data/json/items/containers.json

```diff
@@ -271,7 +271,7 @@
         "description": "A standard plastic jug used for milk and household cleaning chemicals.",
         "price": 0,
         "weight": 190,
-        "volume": 15,
+        "volume": 18,
```

Comment:
I'd prefer `volume == capacity`. I wasn't aware that there was a specific US gallon.